src/SessionOne.java

```diff
@@ -1,54 +1,79 @@
 import java.util.Scanner;
 
 public class SessionOne {
+
     public static void main(String[] args) {
         Scanner input = new Scanner(System.in);
-        System.out.println("Press 1, if you want to see if it's an SessionOne number");
+        System.out.println("Press 1, if you want to see if it's an Armstrong number");
         System.out.println("Press 2, whether you want to see if it's an Strong number");
+        System.out.println("Press 3, whether you want to see if it's an Perfect number");
+
         int option = input.nextInt();
+
         System.out.println("Enter a number");
         String number = String.valueOf(input.nextInt());
 
-            switch (option) {
-                case 1:
-                    isAnAmstrongNumber(number);
-                case 2:
-                    isAnStrongNumber(number);
-            }
+        switch (option) {
+            case 1:
+                isAnArmstrongNumber(number);
+            case 2:
+                isAnStrongNumber(number);
+            case 3:
+                isAPerfectNumber(Integer.valueOf(number));
         }
+    }
 
-        public static void isAnAmstrongNumber(String number) {
-            String[] digits = number.split("(?<=.)");
-            double is_amstrong = 0;
-            for (int i = 0; i < digits.length; i++) {
-                is_amstrong += Math.pow(Double.parseDouble(digits[i]), 3);
-            }
-            if (Integer.valueOf(number) == is_amstrong) {
-                System.out.println("it is an SessionOne number");
-            } else {
-                System.out.println("it is not an SessionOne number");
-            }
+    public static void isAnArmstrongNumber(String number) {
+        String[] digits = number.split("(?<=.)");
+
+        double isArmstrong = 0;
+
+        for (int i = 0; i < digits.length; i++) {
+            isArmstrong += Math.pow(Double.parseDouble(digits[i]), 3);
         }
 
-        public static void isAnStrongNumber(String number){
-            String[] digits = number.split("(?<=.)");
-
-            double isStrong = 0;
-            for (int i = 0; i < digits.length; i++) {
-                int factorial = 1;
-                int digit = Integer.valueOf(digits[i]);
-                for (int j = digit; j > 1; j=j-2) {
-                    int tempNumber = 0;
-                    tempNumber = (j * (j-1));
-                    factorial *= tempNumber;
-                }
-                isStrong +=factorial;
-            }
-            if(isStrong == Integer.valueOf(number)){
-                System.out.println("The number is a Strong number "+isStrong);
+        if (Integer.valueOf(number) == isArmstrong) {
+            System.out.println("it is an Armstrong number");
+        } else {
+            System.out.println("it is not an Armstrong number");
+        }
+    }
+
+    public static void isAnStrongNumber(String number) {
```

Comment:
Apparently the conversation got pretty long the last time, and you forgot this one. (However I expect you to have read the comments that were left there) So I will bring this up again. It should be `isAStrongNumber` ,`S` is not a vowel.
```suggestion
    public static void isAStrongNumber(String number) {
```